solr/solrj/src/java/org/apache/solr/client/solrj/impl/BaseHttpClusterStateProvider.java

```diff
@@ -228,12 +249,9 @@ > getCacheTimeout()) {
   }
 
   @SuppressWarnings({"rawtypes", "unchecked"})
-  private static Set<String> fetchLiveNodes(SolrClient client) throws Exception {
-    ModifiableSolrParams params = new ModifiableSolrParams();
-    params.set("action", "CLUSTERSTATUS");
-    QueryRequest request = new QueryRequest(params);
-    request.setPath("/admin/collections");
-    NamedList cluster = (SimpleOrderedMap) client.request(request).get("cluster");
+  private Set<String> fetchLiveNodes(SolrClient client) throws Exception {
+
+    SimpleOrderedMap<?> cluster = submitClusterStateRequest(client, null, true, false, false);
```

Comment:
oh yes, null, true false false.   Uh..... what does that refer to? ;-).    See my other comment.

Reply:
Addressed in latest commit